resources/application-connector/charts/application-operator/values.yaml

```diff
@@ -9,6 +9,7 @@ controller:
     tillerTLSInsecure: true
     syncPeriod: 30
     installationTimeout: 240
+    gatewayOncePerNamespace: false
```

Comment:
I would suggest moving this value to global values and rename it to something more general, as this override will be used by other components as well.